transact_write_items.go

```diff
@@ -52,32 +64,5 @@ func (e *MockDynamoDB) TransactWriteItems(input *dynamodb.TransactWriteItemsInpu
 }
 
 func (e *MockDynamoDB) TransactWriteItemsWithContext(ctx aws.Context, input *dynamodb.TransactWriteItemsInput, opts ...request.Option) (*dynamodb.TransactWriteItemsOutput, error) {
-	if len(e.dynaMock.TransactWriteItemsExpect) > 0 {
-		x := e.dynaMock.TransactWriteItemsExpect[0] //get first element of expectation
-
-		foundTable := false
-
-		if x.table != nil {
-			for _, item := range input.TransactItems {
-				if (item.Update != nil && x.table == item.Update.TableName) ||
```

Comment:
Not sure if you guys reviewed this or not, but it seems like if one item has the correct table name, but the rest don't, the `foundTable` will still be set to true! And it is not one table, TransactWriteItems can have items that target different tables.